enlighten/Measurement.py

```diff
@@ -233,7 +233,8 @@ class Measurement(object):
                            'Battery %',
                            'Device ID',
                            'FW Version',
-                           'FPGA Version']
+                           'FPGA Version',
+                           'NOTE']
```

Comment:
To me this is weird, because "Note" is already in CSV_HEADER_FIELDS.  Was this a case-sensitivity issue?